.github/workflows/ci-all-testing.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: tests
+name: CI All Test
```

Comment:
just call it tests for now

Reply:
K renamed 👍  just keeping stuff as `Tests` Title looks easier on the eyes.